config/install/views.view.event_invites_and_chat_appointments.yml

```diff
@@ -0,0 +1,826 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.dip_chat_appointment.field_appointmentdate
+    - field.storage.living_spaces_event.field_end_date
+    - field.storage.living_spaces_event.field_start_date
+    - search_api.index.event_invites_and_chat_appointments
```

Comment:
This view as well should be moved to dip_chat_counseling as depends on index from there